data/src/main/java/me/androidbox/data/local/converter/AttendeeConverter.kt

```diff
@@ -0,0 +1,22 @@
+package me.androidbox.data.local.converter
+
+import androidx.room.TypeConverter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import me.androidbox.data.local.entity.AttendeeEntity
+
+class AttendeeConverter {
+    private val moshi = Moshi.Builder().build()
```

Comment:
You could receive the moshi in the constructor, and inject it with HIlt

Reply:
Yes, I was thinking of doing that.